teamA/kimTaeksoo/controller/userController.js

```diff
@@ -0,0 +1,19 @@
+const userService = require("../service/userService");
+
+const signUp = async (req, res) => {
+  const { name, email, profileImage, password } = req.body;
+  try {
+    if (!name || !email || !profileImage || !password) {
+      return res.status(400).json({ message: "fail" });
```

Comment:
요청의 body는 key와 value로 구성된 JSON 형태를 가집니다.

해당 코드는 key에 따른 value 여부를 판단하는 코드로서 반환하는 에러 메시지에도 해당 내용이 반영되도록 수정해주세요!

```suggestion
      return res.status(400).json({ message: "Key Error" });
```